tests/run_local_job.py

```diff
@@ -0,0 +1,12 @@
+# This is meant as a simple test file to check if psi/j was installed successfully
+
+from psi.j import SubmitException, Job, JobExecutor, JobSpec, JobState
```

Comment:
Some imports are not used which probably will trip a linter.

Reply:
Good point. The tests were not even checked.

I added the tests dir to "make checks" and removed the unused imports.